### DISCUSSION TITLE
About MobileNet_V2 training config

### DISCUSSION THREAD

**Discussion by jkboyJohn** [STVIR/pysot#525] [Q&A]
I wanna reproduce MobileNet_V2 tracking results, now I found the pretrained model of MobileNet_V2 from torchvision modles,  is it right? And could you tell me your official training  parameter configuration?
Thank you

**Comment by ShuKe-code** [comment]
I'm training MobileNet_V2, too.But the effect is not good.Have you solved the problem？
